views/component/filterSummary/filterSummaryRow/template.njk

```diff
@@ -0,0 +1,19 @@
+{% macro renderFilterSummaryTag(tag, submitUrl) %}
+    <input type="hidden" name="{{tag["formName"]}}" id="{{tag["formName"]}}-{{tag["formValue"]}}" value="{{tag["formValue"]}}">
+    {% if tag["preText"] %}
+        <span class="filter-summary__tag-pretext">{{tag["preText"]}}</span>
+    {% endif %}
+    <div class="filter-summary__tag"><span>{{ tag["tagText"] }}</span>{% if tag["dismissable"] %}<button formaction="{{ submitUrl }}?remove={{tag["formName"]}},{{tag["formValue"]}}" type="submit" class="filter-summary__tag-dismiss">X</button>{% endif %}</div>
+{% endmacro %}
+
+<div class="filter-summary__row">
+    <div class="filter-summary__tags">
```

Comment:
Wondering if it's worth replacing these `<div>`s for the filters with `<section>`s and a visually hidden heading.

Reply:
Updated in `views/component/filterSummary/template.njk`